vent/api/actions.py

```diff
@@ -177,7 +177,16 @@ def system_commands():
        return
 
    @staticmethod
-   def logs():
+   def logs(grep_list=None):
+       # generically filter logs stored in log containers
+       cores = ["core-aaa-syslog", "core-aaa-redis", "core-aaa-rabbitmq", "core-rmq-es-connector", "vent-management"]
+       if grep_list:
+           for expression in grep_list:
+               for core in cores:
+                   os.system("docker logs "+core+" | grep "+expression+" | tee")
```

Comment:
I added the `docker-py` library to work natively with the vent package, so we should use that for docker commands to make it less brittle in the future.

check this out for how to set up the environment:

https://github.com/CyberReboot/vent/blob/experimental/vent/menu.py#L10